che_plugin_brokers.adoc

```diff
@@ -90,34 +98,43 @@ repository: https://www.sonarlint.org/
 ----
 
-TIP: `containerImage` attribute specifies the image that would be used for running the plugin as a sidecar container. If the `containerImage` attribute is not specified, the plugin will be running inside the main Theia container.
+[NOTE]
+====
+The `containerImage` attribute specifies the image that is used for running the plug-in as a sidecar container. If the `containerImage` attribute is not specified, the plug-in runs inside the main Theia container.
+====
 
-More information about the plugin registry can be found in the README.md of the https://github.com/eclipse/che-plugin-registry[che-plugin-registry] repository.
+For more information about the plug-in registry, see the `README.md` file of the https://github.com/eclipse/che-plugin-registry[che-plugin-registry] repository.
 
-== Plugin Brokers
-Now, when the `Plugin` and `Plugin Registry` have been discussed, it is high time to introduce the `Plugin Broker` concept.
+[id='plugin-brokers']
+== Plugin brokers
 
-**Plugin Broker** - is an application that runs just before the actual workspace start phase and based on the received plugin metadata deliveries the plugin binaries to the workspace containers, and sends workspace configuration changes to the `Che Server` that are needed to be applied before the workspace startup.
+The *Plugin Broker* application runs immediately prior to the workspace start phase. Based on the received plug-in metadata, the *Plugin Broker* application delivers the plug-in binaries to the workspace containers. It sends workspace configuration changes to the Che server. These configuration changes should be applied before the workspace is started.
 
-Taking into account, that there are multiple types of plugins, there are also multiple types of brokers since each type of plugin needs to be processed differently:
+Each plug-in must be processed differently and hence there are multiple types of plug-ins and brokers.
 
-* `Che Plugin Broker` for `Che Plugin` and `Che Editor` types
-* `VS Code extension Broker` for `VS Code extension Plugin` type
-* `Theia Plugin Broker` for `Theia Plugin` type
+* The Che plug-in broker for the Che plug-in and the Che editor types
+* The VS Code extension broker for the VS Code extension plug-in type
+* The Theia plug-in broker for the Theia plug-in type
 
-TIP: Implementation details and differences of the existing plugin brokers can be found in the  `README.MD` of the dedicated https://github.com/eclipse/che-plugin-broker[che-plugin-broker] repository.
+For implementation details and differences in the existing plug-in brokers, see the `README.MD` file of the dedicated https://github.com/eclipse/che-plugin-broker[che-plugin-broker] repository.
 
-CAUTION: Since Che supports pluggable editor implementations (Theia, Dirigible, Jupyter Notebook), it is very likely that the new types of plugins with the dedicated brokers would appear in future e.g. `Dirigible Plugin` and the corresponding `Dirigible Plugin Broker`.  
```

Comment:
@supriya-bharadwaj why have you decided to remove adoc `CAUTION:` section

Reply:
@ibuziuk , this sentence, again, looked more like a general statement to me. Do we have any reason to "caution" the user about things appearing in the upcoming releases? Also, when those releases happen and things actually change, we can, at that time, let the user know that this has changed. WDYT?